### PR TITLE
ported mjpeg video support from openHAB 1

### PR DESCRIPTION
**Pull request by kaikreuzer**
See https://community.openhab.org/t/video-in-sitemap-breaks-ui/8542

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: bundles/model/org.eclipse.smarthome.model.sitemap/src/org/eclipse/smarthome/model/Sitemap.xtext [hunk 1]
```diff
@@ -45,11 +45,13 @@ Image:
...
 		('labelcolor=[' (LabelColor+=ColorArray (',' LabelColor+=ColorArray)* ']'))? &
-		('valuecolor=[' (ValueColor+=ColorArray (',' ValueColor+=ColorArray)* ']'))? &		('iconcolor=[' (IconColor+=ColorArray (',' IconColor+=ColorArray)* ']'))? &
+		('valuecolor=[' (ValueColor+=ColorArray (',' ValueColor+=ColorArray)* ']'))? &
+		('iconcolor=[' (IconColor+=ColorArray (',' IconColor+=ColorArray)* ']'))? &
 		('visibility=[' (Visibility+=VisibilityRule (',' Visibility+=VisibilityRule)* ']'))?);
 
 Video:
-	'Video' (('item=' item=ItemRef)? & ('label=' label=(ID | STRING))? & ('icon=' icon=Icon)? & ('url=' url=(STRING)) &
+	'Video' (('item=' item=ItemRef)? & ('label=' label=(ID | STRING))? & ('icon=' icon=Icon)? & 
+	    ('url=' url=(STRING)) & ('encoding=' encoding=(STRING))? &
```
Comment (maggu2810) [inline bundles/model/org.eclipse.smarthome.model.sitemap/src/org/eclipse/smarthome/model/Sitemap.xtext]: Wrong indent (mix of spaces and tabs)? Or don't we care about mixing indent for Xtext files?

**Comment by maggu2810** [comment]
Only one comment about indentation.

**Comment by kaikreuzer** [comment]
I have added a second commit which applies the code formatter on the complete file. Probably better keep the two commits in place when merging for better traceability.

**Comment by maggu2810** [comment]
merged without squashing

**Pull request merged**